products/compute/api.yaml

```diff
@@ -1256,60 +1253,282 @@ objects:
       error: !ruby/object:Api::Async::Error
         path: 'error/errors'
         message: 'message'
-    properties:
-      - !ruby/object:Api::Type::String
-        name: 'name'
-        required: true
-        input: true
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'region'
+        resource: 'Region'
+        imports: 'name'
         description: |
-          Name of the resource. Provided by the client when the resource is
-          created. The name must be 1-63 characters long, and comply with
-          RFC1035. Specifically, the name must be 1-63 characters long and match
-          the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the
-          first character must be a lowercase letter, and all following
-          characters must be a dash, lowercase letter, or digit, except the last
-          character, which cannot be a dash.
-      - !ruby/object:Api::Type::Array
-        name: 'healthChecks'
-        item_type: Api::Type::String
+          A reference to the region where the regional backend service resides.
         required: true
-        min_size: 1
-        max_size: 1
+    properties:
+      - !ruby/object:Api::Type::Integer
+        name: 'affinityCookieTtlSec'
```

Comment:
Should this be beta-only since it's only used when the BS is INTERNAL_SELF_MANAGED?